### PR TITLE
#90 [UX/UI] Icon overlaps "Buy Tickets" button on mobile view

### PR DESCRIPTION
**Pull request by withshubh**
updated media queries in the responsive.css file. No changes in the desktop version of the website only mobile version updated.

### CODE REVIEW

**Comment by sagarchoudhary96** [comment]
@withshubh Is this issue fixed or need to be looked?

**Comment by withshubh** [comment]
FIxed

**Comment by sagarchoudhary96** [comment]
great :+1:

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please consider following changes.
* The changes you're making are on the responsive.css which is used by the entire website. You should always add or modify main.css since it contains page/element specific styles.
* You are targeting all the .tg-btn here to modify just one Ticket button. Instead of this use is or class specific to that particular button only.

File: index.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h1 class="animated zoomIn">PyCon India 2017</h1>
 									</li>
 								</ul>
 								<div class="tg-btnarea">
-									<a class="tg-btn tg-btnbookseat" style="color: #FFF;" href="https://in.explara.com/e/pycon-india-2017" rel="noopener noreferrer">Buy Tickets</a>
+									<a class="tg-btn tg-btnbookseat" style="color: #FFF;max-width: 80%" href="https://in.explara.com/e/pycon-india-2017" rel="noopener noreferrer">Buy Tickets</a>
```
Comment (realslimshanky) [inline index.html]: Don't use inline style. Transfer these styles to main.css

**Review by realslimshanky — CHANGES_REQUESTED** [review]

**Comment by realslimshanky** [comment]
@withshubh bro, 
* you have made change in the first commit and then revoked that change in the second.
* Then main some changes again in the third commit revoked that too in fourth.
* Made changes to index.html on fifth commit and revoked that change when I suggested you should transfer the style on stylesheet but you haven't added the styles yet. 
Feel free to ask for any assistance if you need.

**Comment by CuriousLearner** [comment]
Is there any update on this @withshubh ? Will you able to do it?

**Comment by withshubh** [comment]
@CuriousLearner "icon overlapping" bug from the website is already removed

**Comment by realslimshanky** [comment]
Closing this PR since the same issue has already been fixed in #95

**Pull request closed without merging**